File: src/app/interfaces/receipt-items-approval-dto.ts
export interface ReceiptItemsApprovalDTO {
    receiptItemId: string;
    receiptId: string;
    itemDescription: string;
    quantity: number;
    unitPrice: number;
    discount: number;
    total: number;
    subCategoryId?: number  ;
    unitOfMeasureId?: number;
}

File: API/FunctionAppDoc2Data/Models/ReceiptItemsApprovalDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FunctionAppDoc2Data.Models;
public class ReceiptItemsApprovalDTO
{
    public Guid ReceiptItemId { get; set; }
    public Guid ReceiptId { get; set; }
    public string ItemDescription { get; set; }
    public decimal Quantity { get; set; }
    public decimal UnitPrice { get; set; }
    public decimal Discount { get; set; }
    public decimal Total { get; set; }
    public int SubCategoryId { get; set; }
    public int UnitOfMeasureId { get; set; }
}
